books/RayTracingTheNextWeek.html

```diff
@@ -2256,7 +2256,319 @@
 
 
 
-Rectangles and Lights
+Quadrilaterals
+====================================================================================================
+We've managed to get more than half way through this three-book series using spheres as our only
+geometric primitive. Time to add our second primitive: the quadrilateral.
+
+
+Ray-Plane Intersection
+-----------------------
+Spheres are generally the first ray tracing primitive because their implicit formula makes it so
+easy to solve for ray intersection. Like spheres, planes also have an implicit formula, and
+ray-plane intersection is even _easier_ to solve.
+
+You may already know this implicit formula for a plane:
+
+  $$ Ax + By + Cz + D = 0 $$
+
+That is, a plane is the set of all points $\mathbf{P} = (x,y,z)$ that satisfy the formula above. It
+makes things slightly easier to use the alternate formulation:
+
+  $$ Ax + By + Cz = D $$
+
+(We didn't flip the sign of D because it's just some constant that we'll figure out later.)
+
+Here's another way to think of this formula: given the plane perpendicular to the normal vector
```

Comment:
Here's a good way to think of this formula:

I don't know if the definition above counts as one way to think of the formula, rather it defines it, so the following is really the _first_ way of thinking about it.

This feels slightly semantic, but that struck out to me.

Reply:
Reworked.